File: senior/MT1051/week04/HW11/HW11_109601003.go
package main

import "fmt"

func main() {
	for i, j := 2, 1; j < 10; {
		tmp := i * j
		fmt.Printf("%dx%d", i, j)
		if tmp >= 10 {
			fmt.Printf("=")
		} else {
			fmt.Printf("= ")
		}
		if i == 9 {
			fmt.Printf("%d\n", i*j)
		} else {
			fmt.Printf("%d ", i*j)
		}

		if i == 9 {
			i = 2
			j++
		} else {
			i++
		}
	}
}

File: mt/mt1051/week04/HW11/cpp/HW11_109601003.cpp
#include <iostream>
using namespace std;

int main(){
for (int i = 2, j = 1;j < 10;(i==9)?(i=(++j/j)+1):(i++))
{ 
 
int tmp = i * j; 
cout << i << "*" << j; 
cout << ((tmp>=10)? "=": "= ") ; 
if(i == 9){cout << i*j << endl;} //hi
else
{ {
 cout << i*j << " "; }

}
} 
 
return 0;
}